src/test/java/gov/healthit/chpl/aqa/asserts/DpmanagementUploadPageAsserts.java

```diff
@@ -36,4 +55,16 @@ public void verifyLongTestTaskIdAndParticipantIdErrorsOnUpload(final String erro
         assertTrue(actualString.contains(errorMessage), "Expect \"" + errorMessage + "\" to be found in \"" + actualString + "\"");
     }
 
+    /**
+     * Load listing details to verify surveillance was uploaded successfully.
+     */
+    @Then("^I see that surveillance was uploaded successfully to CHPL and listing details show surveillance activity$")
+    public void verifySurveillanceConfirmWasSuccessful() {
+        WebElement accordion = ListingDetailsPage.surveillanceActivitiesAccordion(getDriver());
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", accordion);
+
+        String surveillanceText = ListingDetailsPage.surveillanceActivitiesPanel(getDriver()).getText();
+
+        assertTrue(surveillanceText.contains("Open Surveillance, Began Sep 20, 2017: 1 Open and 2 Closed Non-Conformities Were Found"));
```

Comment:
How will we know that the surveillance found here isn't the one that was uploaded "the day before yesterday", or whatever?

Reply:
I had added a note when creating a PR, along with commit message but it's not showing. Next to do is delete uploaded surveillance from this listing so there's no previous surveillance activity with same dates and NC counts. manageSurveillance page is cached and surveillance activity doesn't load/show unless I open a new session in different browser, cache refresh in same session didn't help either. I need to figure out a way to access it in same session manually first. Any help/pointers to get that working? that way 'surveillance delete' would also be covered.

Reply:
Is there something in the surveillance workflow you could edit during the upload, like we do with the listing's Product & Version codes? The "number of sites surveilled" could be any integer, so if you used the day of the year (1-365) it'd likely never duplicate, as we do delete these surveillances every time the STG plan runs. Could also use one of the dates, though that restricts us to once per day

Reply:
I'm confused about what is cached regarding pending surveillance. The API call to get pending surveillance is not cached so I think any time you view the list of pending surveillance it should return a fresh list. Could you just click all the checkboxes next to each pending surveillance on the page and then the Reject All button to remove them after upload?

Reply:
not the pending surveillance page that's cached, manageSurveillance page where you could load surveillance for a listing and update/delete